content/en/blog/2024/ambient-vs-cilium/index.md

```diff
@@ -0,0 +1,67 @@
+
+{{< image width="80%"
+    link="./scale-scorecard.png"
+    alt="Scalability Scorecard: Istio vs. Cilium!"
+    >}}
+
+* **The Cilium Slowdown:** Cilium, while boasting impressive low latency with default install parameters, slows down substantially when Istio’s baseline features such as L7 policy and encryption are turned on. Additionally, cilium’s memory and CPU utilization remained high even when no traffic was flowing in the mesh. This can impact the overall stability and reliability of your cluster, especially as it grows.
+* **Istio Ambient Mode: The Steady Performer:** Istio's Ambient mode, on the other hand, showed its strength in stability and maintaining decent throughput, even with the added overhead of encryption. While Istio did consume more memory and CPU than Cilium under test, its CPU utilization settled to a fraction of Cilium’s when not under load.
+
+## Behind the Scenes: Why the Difference?
+
+The key to understanding these performance differences lies in the architecture and design of each tool.
+
+* **Cilium's Control Plane Conundrum:** Cilium runs a control plane instance on each node, leading to API server strain and configuration overhead as your cluster expands. This frequently caused our API server to crash, followed by Cilium becoming unready, and the entire cluster becoming unresponsive.
+* **Istio's Efficiency Edge:** Istio, with its centralized control plane and identity-based approach, streamlines configuration and reduces the burden on your API server and nodes, directing critical resources to processing and securing your traffic, rather than processing configuration. Istio takes further advantage of the resources not used in the control plane by running as many Envoy instances as a workload needs, while Cilium is limited to one shared Envoy instance per node.
+
+## Digging Deeper
+
+While the objective of this project is to compare Istio and Cilium scalability, several constraints make a direct comparison difficult.
+
+### Layer 4 Isn’t always Layer 4
+
+While Istio and Cilium both offer L4 policy enforcement, their APIs and implementation differ substantially. Cilium implements Kubernetes NetworkPolicy, which uses labels and namespaces to block or allow access to and from IP Addresses. Istio offers an AuthorizationPolicy API, and makes allow and deny decisions based on the TLS identity used to sign each request. Most defense-in-depth strategies will need to make use of both NetworkPolicy and TLS-based policy for comprehensive security.
+
+### Not all Encryption is Created Equal
+
+While Cilium offers IPsec for FIPS-compatible encryption, most other Cilium features such as L7 Policy and Load Balancing are incompatible with IPsec. Cilium has much better feature compatibility when using WireGuard encryption, but WireGuard cannot be used in FIPS-compliant environments. Istio, on the other-hand, uses FIPS-compliant mTLS by default, which has the added benefit of complying with Zero-Trust principles.
```

Comment:
```suggestion
While Cilium offers IPsec for FIPS-compatible encryption, most other Cilium features such as L7 Policy and Load Balancing are incompatible with IPsec. Cilium has much better feature compatibility when using WireGuard encryption, but WireGuard cannot be used in FIPS-compliant environments. Istio, on the other-hand, because it strictly complies with TLS protocol standards, always uses FIPS-compliant mTLS by default.
```

`which has the added benefit of complying with Zero-Trust principles.` reads as vague/handwavy opinion to me. That may be the intent, in which case ignore.

It might also be clearer and more readable to just change the comma to a period and drop the coda, eg:

```
Istio, on the other-hand, uses FIPS-compliant mTLS by default.
```